lib/install.js

```diff
@@ -46,6 +46,11 @@ function install(opts, cb) {
     opts.drivers = defaultConfig.drivers;
   }
 
+  if (process.platform !== 'win32') {
+    delete opts.drivers.ie;
```

Comment:
Hi! What's the difference between delete opts.drivers.ie and deleting fsPaths/urls. I am unsure about the consequences between the two. Since you worked on this, do you have more details?

Thanks, this looks nice